packages/core/phase2/lib/getOperationSequence/areAllResultsAlreadyPresentOnPnc/isMatchToPncAdjAndDis/createDisposalByResult/validateResultQualifierVariableDurationType.ts

```diff
@@ -0,0 +1,23 @@
+import ExceptionCode from "bichard7-next-data-latest/dist/types/ExceptionCode"
+import errorPaths from "../../../../../../lib/exceptions/errorPaths"
+import type { Result } from "../../../../../../types/AnnotatedHearingOutcome"
+import type Exception from "../../../../../../types/Exception"
+
+const validateResultQualifierVariableDurationType = (
+  result: Result,
+  offenceIndex: number,
+  resultIndex: number
+): Exception[] =>
+  result.ResultQualifierVariable?.reduce((acc: Exception[], qualifierVariable, qualifierVariableIndex) => {
+    if (qualifierVariable.Duration?.DurationType !== undefined) {
+      acc.push({
+        code: ExceptionCode.HO200201,
+        path: errorPaths.offence(offenceIndex).result(resultIndex).resultQualifierVariable(qualifierVariableIndex)
+          .DurationType
+      })
+    }
+
+    return acc
+  }, [])
```

Comment:
Sorry another Ting nitpick, not a fan of this pattern we have of using `acc` when using `reduce()` over a more meaningful accumulator name.

```suggestion
  result.ResultQualifierVariable?.reduce((exceptions: Exception[], qualifierVariable, qualifierVariableIndex) => {
    if (qualifierVariable.Duration?.DurationType !== undefined) {
      exceptions.push({
        code: ExceptionCode.HO200201,
        path: errorPaths.offence(offenceIndex).result(resultIndex).resultQualifierVariable(qualifierVariableIndex)
          .DurationType
      })
    }

    return exceptions
  }, [])
```

Reply:
Makes sense. 👍
Renamed. ✔️